lms/static/scripts/frontend_apps/services/grading.ts

```diff
@@ -6,6 +6,9 @@ export type Student = {
 
   /** API URL for posting outcome results. */
   LISOutcomeServiceUrl: string;
+
+  /** LTI user id for this student. */
+  lmsId: string;
```

Comment:
This value was already part of the data sent by the BE:

https://github.com/hypothesis/lms/blob/main/lms/resources/_js_config/__init__.py#L321